setup.py

```diff
@@ -46,6 +46,7 @@ def prerelease_local_scheme(version):
         'histomicsui',
         'jsonschema',
         'large_image[tiff,openslide,memcached,openjpeg]',
+        'large_image_converter',
```

Comment:
Do we need to add this?

Reply:
I don't think we need this to redact areas from associated images. However I was running into problems when going through the workflow to redact areas of the whole slide image from the main image viewer. After selecting an area to redact and clicking the `Redact` button, I received this error message: "Failed to redact image. No module named 'large_image_converter."

I think the only place it's used is in `process.py::read_ts_as_vips`, but from what I can tell, it is needed, at least for this. Do you want me to break that out into a separate PR?

For completeness, I'm doing this testing in a local docker container, following the instructions in the install docs to create an image from my local source code.